user-operator/src/main/java/io/strimzi/operator/user/model/KafkaUserModel.java

```diff
@@ -60,6 +60,8 @@
 
     public static final String KAFKA_USER_OPERATOR_NAME = "strimzi-user-operator";
 
+    public static final String K8S_SECRET_PREFIX = System.getenv("K8S_SECRET_PREFIX");
```

Comment:
I think the environment variable should be extracted and validated in `UserOperatorConfig` class to have it together with the other configs. And it should be passed to the model through the `KafkaUserOperator`.